app/src/main/res/values/strings.xml

```diff
@@ -242,41 +242,43 @@
     <string name="login_enter_encryption_details">Secret Encryption Password</string>
     <string name="login_encryption_account_label">Account:</string>
     <string name="login_service_details_description">This is your login password, *not* your encryption password!</string>
-    <string name="login_forgot_password">Forgot password?</string>
+    <string name="login_forgot_password">Forgot your password?</string>
 
-    <string name="login_wrong_username_or_password">Couldn\'t authenticate (probably wrong username or password).\nHave you registered?</string>
+    <string name="login_configuration_detection">Setup detection</string>
+    <string name="login_querying_server">Please wait, querying server…</string>
```

Comment:
What are these strings you added? Maybe a wrong merge?

Reply:
I'll have a look to see what happened.